ui/page/transaction/transactions_page.go

```diff
@@ -222,95 +230,127 @@ func (pg *TransactionsPage) loadTransactions(offset, pageSize int32) (interface{
 // Layout draws the page UI components into the provided layout context
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
-func (pg *TransactionsPage) Layout(gtx layout.Context) layout.Dimensions {
+func (pg *TransactionsPage) Layout(gtx C) D {
 	if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
 		return pg.layoutMobile(gtx)
 	}
 	return pg.layoutDesktop(gtx)
 }
 
-func (pg *TransactionsPage) layoutDesktop(gtx layout.Context) layout.Dimensions {
+func (pg *TransactionsPage) pageLayout(gtx C, body layout.Widget) D {
```

Comment:
`pg.pageLayout` stutters (pg/page is repeated). Also, did you mean `layoutPage`? There's already a `pg.Layout()` method, so whatever this does, rename the method and add a descriptive comment.